examples/gettingStarted/gettingStarted.ino

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
Should the file and directory be named with an upper case, like ```GettingStarted/GettingStarted.ino```?

Reply:
This may be getting into personal preference... I/we should review the NASA C style guide first (link in CONTRIBUTING.md) 😆 For the RF24 lib's new examples, I tried to use a lowercase first letter and camel case the rest for the Arduino examples. For the RF24 Linux examples, I went with first letter uppercase... And then there's python for which PEP-8 prefers all lowercase letters with underscores as word delimiters (for practically everything except class names).

Reply:
I just skimmed the NASA C style guide and it deems almost all usual casing as acceptable.

I did notice that it doesn't actually specify a convention for filenames (other than literally saying "we recommend using longer, more explicit names").
![image](https://user-images.githubusercontent.com/14963867/117376405-e91ff080-ae85-11eb-858c-d3b7acc10e67.png)

Like I said before, I tend to use different casing for almost identical examples to try and differentiate between which platform the example was written for; this is for the developer's benefit, not the user's benefit.

Reply:
>this is for the developer's benefit, not the user's benefit

I'm sorry but in my few sentences before I was using **user** word in the meaning of **developer**. This could cause some misunderstandings. Since now I will just use **developer** when I really mean a developer.

Reply:
All files'/examples' names now use uniform casing